src/checks/storage_zfcp.ts

```diff
@@ -0,0 +1,23 @@
+import { it, page } from "../lib/helpers";
+import { SidebarPage } from "../pages/sidebar_page";
+import { StoragePage } from "../pages/storage_page";
+import { SelectInstallationDevicePage } from "../pages/select_installation_device_page";
+import { ZfcpPage } from "../pages/zfcp_page";
+
+export function prepareZfcpStorage() {
+  it("should prepare ZFCP storage", async function () {
```

Comment:
Do you know that you can do suggestions like that? For better readability.

```suggestion
  it("should prepare zFCP storage", async function () {
```